README.md

```diff
@@ -114,3 +122,12 @@ The scripts are written to work with [A-grid ocean data from the Copernicus Mari
 ## Source code
 
 The code for this project is [hosted on GitHub](https://github.com/OceanParcels/virtualship).
```

Comment:
Can we also have a GitHub logo on the top-right, like in https://docs.oceanparcels.org/en/latest/index.html?

Reply:
This uses a different theme which, unfortunately, doesn't allow the customisation of these buttons. Perhaps we can add a link ("Project on GitHub") to the left sidebar? There is already a github link in the footer, though that is admittedly out of the way.

Reply:
OK clear. But why don’t we use the same theme as Parcels itself? Is there a good reason for that? One theme would be nice for consistency/recognition, and would also reduce maintenance (a bit)?

Reply:
> Is there a good reason for that?

Nope

> One theme would be nice for consistency/recognition, and would also reduce maintenance (a bit)?

Yes, sounds good. I can work on a migration